### DISCUSSION TITLE
don't appear in pro tools menu Mac

### DISCUSSION THREAD

**Discussion by oudileoudi** [apohl79/audiogridder#1031] [Help] [answered]
hello 
first thank you for you job.....really really nice
but
I've change my computer with a new m1 MacBook Pro.
and I've installe all version of you plug but the plug don't appear in protools plug menu.
in ly computer in can see that the plug is on the good place.
and with bluecoat patchwork when I try to open it I've a message 
Could not load "AudioGridder"

Please check that this is a compatible 64-bit Intel Audio Unit plug-in.

but look work with reaper.....

thank you in advance for your help, and sorry for my bad English.
best
jf

**Comment by apohl79** [comment, marked as answer]
For pro tools you need the intel version of AG (you can also install the universal version), also on an M1 mac, as pro tools does not support ARM yet and runs under rosetta.

**Comment by oudileoudi** [comment]
thks for you're help.
but I can't not change pro tools 22.7.0.105 to rosetta I don't have the choice .
or I don't know how :-)
in other app in about app I can choose rosetta not pro tools.
thk.
jf

**Reply by apohl79** [reply]
I guess you only have the choice, if the app is a universal binary (that includes an executable for intel and arm). But as PT is intel only, it will always run under rosetta I guess.

**Comment by oudileoudi** [comment]
ok thank you again....
I've not understand that I MUST INSTAL the intel chip .....
and you told me :-)
it look that my plug appear now.
thank you for your help.
best .
jf
